files/ja/web/html/element/input/file/index.md

```diff
@@ -323,9 +325,9 @@ function updateImageDisplay() {
       const listItem = document.createElement("li");
       const para = document.createElement("p");
       if (validFileType(file)) {
-        para.textContent = `ファイル名: ${file.name}, ファイルサイズ: ${returnFileSize(
-          file.size
-        )}.`;
+        para.textContent = `ファイル名: ${
```

Comment:
This line shouldn't be broken, too.

Reply:
Prettier is putting the line break in an odd spot...  Setting `printWidth: 100` in the Prettier configuration seems to bypass this and retain formatting that's similar to where the line break is in upstream.  Maybe Prettier is treating Kanji as double-width characters or something like that?

(In this particular case, though, I don't think that it matters much; the break does not actually affect the output itself.)